tasks/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
 - name: Install unzip
-  apt: pkg=unzip={{ unzip_version }} state=present
+  apt: pkg=unzip={{ unzip_version }} state=present update_cache=yes
```

Comment:
I think `update_cache` should be removed from here and instead go in the `site.yml` in the `examples` directory. If we keep it here, anyone who uses the roll will incur the wait associated with an `apt-get update` every time this task gets executed.

Example:

```yaml
```